neurodsp/sim/cycles.py

```diff
@@ -230,6 +236,10 @@ def sim_gaussian_cycle(n_seconds, fs, std):
         Sampling frequency of the cycle simulation.
     std : float
         Standard deviation of the gaussian kernel, in seconds.
+    center : float, optional, default: 0.5
+        The center of the gaussian.
```

Comment:
This definition isn't super clear (it's not obvious if it's about x or y axis), and I think the description in `sim_skewed_gaussian` is clearer and could be copied here.